app/javascript/controllers/proposal_form_controller.js

```diff
@@ -108,4 +107,12 @@ export default class extends Controller {
       }
     }
   }
+
+  renderPdf (proposalId) {
+    $.post("/proposals/" + proposalId + "/latex",
+      { latex: 'all' },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/birs-math/proposals/pullRequest?prid=7656913)
```suggestion
          { latex: "all" },
```